lib/src/mt_dev.c

```diff
@@ -1767,6 +1775,13 @@ struct mt_tx_queue* mt_dev_get_tx_queue(struct mtl_main_impl* impl, enum mtl_por
 
   mt_pthread_mutex_lock(&inf->tx_queues_mutex);
   for (uint16_t q = 0; q < inf->max_tx_queues; q++) {
+    if (inf->drv_type == MT_DRV_IGC && inf->tx_pacing_way == ST21_TX_PACING_WAY_TSN) {
+      if (is_st21_traffic) {
+        if (q != 0) break;
```

Comment:
This break will break the for loop, and then goto the failed path. Is it we need?

Reply:
> This break will break the for loop, and then goto the failed path. Is it we need?

Yes. Because queue 0 is the only choice of SMPTE 2110 traffic in TSN pacing mode, if queue 0 is NOT available, it means the pool of Tx queue for SMPTE 2110 traffic being empty. So, in such case, no queue available process should be executed.